### PR TITLE
fix: trace incomplete

### PR DESCRIPTION
**Pull request by mscb402**

### CODE REVIEW

File: internal/utils/trace_verbose.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"github.com/api7/cloud-cli/internal/output"
+	"github.com/api7/cloud-go-sdk"
+	"sync"
```
Comment (tokers) [inline internal/utils/trace_verbose.go:20]: Sort the import.

File: internal/utils/trace_verbose.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+func VerboseGoroutine(traceChan <-chan *cloud.TraceSeries) {
+	VerboseWg.Add(1)
+	defer VerboseWg.Done()
+
+	for {
+		select {
+		case data := <-traceChan:
+			DumpTrace(data)
+		default:
+			if WantExit {
```
Comment (tokers) [inline internal/utils/trace_verbose.go:36]: That's a tricky way. Why not use the context to control the exit of this goroutine.

Comment (mscb402) [reply internal/utils/trace_verbose.go]: Done

File: internal/utils/trace_verbose.go (hunk 1 above)
Comment (mscb402) [reply internal/utils/trace_verbose.go:19]: Done

**Review by nic-6443 — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**